### ISSUE TITLE
Failed to build gem native extension

### ISSUE THREAD

**Issue by robbl** [mrrad/heroku-buildpack-taglib#1]
Hey there,

when using the multibuildpack to include the taglib and ruby buildpacks to my rails application setup for heroku, I get the following error after pushing to heroku. 

```
Installing taglib-ruby (0.6.0)
Gem::Installer::ExtensionBuildError: ERROR: Failed to build gem native extension.

/tmp/build_4e928706-e317-4a70-ad26-a850472178ef/vendor/ruby-2.0.0/bin/ruby extconf.rb
checking for main() in -lstdc++... yes
checking for main() in -ltag... no
You must have taglib installed in order to use taglib-ruby.
```

I wonder if I missed something and you could help me out. For some reason taglib is not in the current path, when bundle is installing my gem. Removing taglib-ruby gem from my Gemfile, I can verify that taglib is installed properly under /app/vendor/taglib. Have you had the same error?

**Comment by randomor** [comment]
Hi, Have you figured out the problem? I am having the same issue here. I'm also building using ruby2.0.

**Comment by robbl** [comment]
Nope, sorry! I gave up trying to figure out some working heroku setup for my application...

**Comment by randomor** [comment]
Thanks anyway. Looks like the only way to proceed is to build my own server on digital ocean...

**Comment by a14m** [comment]
stuck in the same issue... anyone had progress solving this ?!

**Comment by fusion2004** [comment]
@artmees I did, though not by modifying the buildpack. I forked taglib-ruby and was able to repoint it to where this buildpack puts it when it builds the gem extensions.
https://github.com/fusion2004/taglib-ruby

Someone else can probably clean up what I did a lot, but it works!

**Comment by a14m** [comment]
@fusion2004 ... first of all thanks i'll try your gem but I want to know what is the problem.
is it that the lib ( which is installed in `/app/vendor/taglib`) is not in the `$PATH` var or what ?

**Comment by fusion2004** [comment]
I believe so, my first attempts including trying to change the PATH env variable on heroku, or trying to change where this buildpack puts taglib. Eventually, my attempts to locate it on the dyno ended up at this commit: https://github.com/fusion2004/taglib-ruby/commit/5cffbd82d6c0aa9fdd03173b331cb1671c54a4e3

Which seems to find taglib properly, so it can build the gem extensions.

**Comment by a14m** [comment]
@fusion2004 thanks for the fix... it works... maybe we can investigate why did it fail with original gem on spare time :D

**Comment by robinst** [comment]
Had a look at this, please comment on robinst/taglib-ruby#28 so it can be solved in the original gem.
